### ISSUE TITLE
Integration still maintained?

### ISSUE THREAD

**Issue by Kugelfang666** [py-smart-gardena/hass-gardena-smart-system#215]
Hi all, 

is this integration still maintained? I’m struggling a lot with it and I see a lot of issues posted but only little progress.

I know all the limitations of hobby projects so no hard feeling at all. At the same time some clarity would be great in order to make a decision between sticking with Gardena equipment or the switch to something else.

**Comment by DNier** [comment]
I asked myself the same question as you. But it really seems like no one is doing anything here anymore. I created a pull request, but no one is looking at it either. It's a pity, but I'll just develop my own thing then.

**Comment by Kugelfang666** [comment]
true and very sad. But currently this integration is borderline-not-working for me. Hence I will likely change my Hardware away from Gardena and to the new SONOFF Water Valves which have just been launched. I Anyhow operate a large Zigbee Network for lighting in my garden so that would be a fit
https://sonoff.tech/product/home-appliances/swv/

The one thing which bends my mind: My understanding ist that Gardena uses the Backend and API of Husqvarna. And there is a native Home Assistant integration for their lawnmowers which also relies on the cloud API. Why isn't the native HA integration also offering the water valves?

**Comment by Kugelfang666** [comment]
just for reference: https://community.home-assistant.io/t/enhancement-of-husqvarna-automower-integration-for-gardena-irrigation-support/745854

**Comment by wolliN59** [comment]
I got the same feeling about missing maintenance for the Gardena Integration. I was delighted when I realized this integration being available via HACS. However, great disappointment followed while trying to configure it and reading these comments. I'm wondering why the developer of this integration does not simply withdraw it instead of guiding people into such a situation ........

**Comment by DNier** [comment]
What components do you have and what exactly isn't working for you? I also only have irrigation components and the Smart Sensors from Gardena, and everything is working with the integration for me.

**Comment by wolliN59** [comment]
Hi DNier, maybe you commented on a post which I already deleted. Anyway, meanwhile I could make it work and I run irrigation control as well as sensors.... seems to work smoothly!

**Comment by Kugelfang666** [comment]
> What components do you have and what exactly isn't working for you? I also only have irrigation components and the Smart Sensors from Gardena, and everything is working with the integration for me.

I have a similar setup.
1x Bridge
1x Sensor 2nd gen
4x Gardena Smart water control. 

For me it’s a lottery whether this integration is properly loaded upon startup of HomeAssitant or not. In most cases it’s not. Sometimes de-activating and re-activating helps, many times it does not. 

Even when it works after a restarted (means I can  open/close the irrigation) It sure stops responding after a few hours.

All In all in the current RC3 version (and also the regular release) this integration has become very unreliable for me.

**Comment by wolliN59** [comment]
Hi,
I'm running a Linux/Home Assistant VM on an unraid server using intel nuc i3 Version 8 HW.
As regards Gardena I' using:
- Gardena smart integration control with 3 valves (subsoil irrigation lawn, water sockets, drip irrigation for bushes and plants)
- 2 sensors for lawn humidity and raised bed humidity

So far I used the smart sw from Gardena, which works quite well.

While trying to integrate almost everything (Z-Wave Network, Tuba smart and Gardena) I got to the point to integrate Gardena with the HACS integration. Due to my own mistakes it took some time to get it up and running.
Currently it seems to work properly. However, I just started and will see whether I will encounter the same problems you mentioned. 

As regards the Z-Wave I kept on using homee (German vendor) with an z-wave cube controller. This worked fine for years. Since the home automation became more complex, I started getting problems. The issue is that the Z-Wave cube is a black box and homee does not provide a HomeKit code to use it as gateway from Home Assistant. Since I'm using an aeotec Z-stick 7 for wave I'm considering to migrate all ZWave  devices to this one and to abandon Homee....

Hope this gives you a small insight into what I'm doing.

If you should have z-Wave experience with HA, feel free to share it with me.

Cheers,
Wolfgang

**Comment by pandalec** [comment]
> > What components do you have and what exactly isn't working for you? I also only have irrigation components and the Smart Sensors from Gardena, and everything is working with the integration for me.
> 
> I have a similar setup. 1x Bridge 1x Sensor 2nd gen 4x Gardena Smart water control.
> 
> For me it’s a lottery whether this integration is properly loaded upon startup of HomeAssitant or not. In most cases it’s not. Sometimes de-activating and re-activating helps, many times it does not.
> 
> Even when it works after a restarted (means I can open/close the irrigation) It sure stops responding after a few hours.
> 
> All In all in the current RC3 version (and also the regular release) this integration has become very unreliable for me.

Same here

**Comment by bpcrocke** [comment]
Same here. Loading is a lottery. And if the integration fails, only a restart works (with manually disabling the integration and manually starting it after system is up and running). This is really bad. Also the mowing time setting under configuration does nothing. But a reliable loading integration with a restart button would be great.

**Comment by grm** [comment]
Hello, 

Sorry for not having too much time too maintain this integration.

I juset merged all opened PR and this coould solve improve this issue.
I am currently trying to have thee time to setup an easy testing environement where i could test the library easily.

If someone is willing to help, we can manage it somehow  !

Grm

**Comment by Kugelfang666** [comment]
Thanks for looking into this, I'll update asap and test in the next days, enjoy the cup of coffee :-)

**Comment by grm** [comment]
Yes, the integration is actively maintained! v2.x has been released with a complete rewrite — new architecture, native websocket, proper auth, LawnMowerEntity support. Please upgrade to v2.x via HACS.
